### PR TITLE
wolfssl: add support for ssl cert blob / ssl key blob options

### PR DESCRIPTION
**Pull request by alexsn**
wolfSSL supports setting certificates/private keys from memory blobs which allow us to implement both CURLOPT_SSLCERT_BLOB and CURLOPT_SSLKEY_BLOB options.

### CODE REVIEW

File: docs/libcurl/opts/CURLOPT_SSLCERT_BLOB.md
Comment (bagder) [inline docs/libcurl/opts/CURLOPT_SSLCERT_BLOB.md:82]: Please use the exact phrase `(added in [version])` for the version information, to allow our automation to work. It can selectively hide this information in the output. The mbedTLS details should be marked up like that as well, the existing way is just an omission.

File: docs/libcurl/opts/CURLOPT_SSLKEY_BLOB.md
Comment (bagder) [inline docs/libcurl/opts/CURLOPT_SSLKEY_BLOB.md:84]: `Added in...`

**Comment by bagder** [comment]
We qualify this as a *change* (not bugfix), which means it needs to wait for the feature window to open before we can merge it. These options will therefore not be supported with wolfSSL in 8.9.0 but most likely in 8.10.0.

**Review by bagder — APPROVED** [review]

**Comment by alexsn** [comment]
> We qualify this as a _change_ (not bugfix), which means it needs to wait for the feature window to open before we can merge it. These options will therefore not be supported with wolfSSL in 8.9.0 but most likely in 8.10.0.

Should I update this pull request to say 8.10.0 in the docs?

**Comment by bagder** [comment]
> Should I update this pull request to say 8.10.0 in the docs?

Yes please!

**Comment by bagder** [comment]
@alexsn can you please fix the merge conflicts in the docs and force-push?

**Pull request closed without merging**
